src/scvmm/HISTORY.rst

```diff
@@ -3,6 +3,17 @@
 Release History
 ===============
 
+0.2.0
```

Comment:
If you don't want to release a new extension, please mark the version in `HISTORY.rst` as `Upcoming`